runtime/compiler/control/J9Options.hpp

```diff
@@ -222,8 +222,9 @@ class OMR_EXTENSIBLE Options : public OMR::OptionsConnector
    static int32_t getScratchSpaceFactorWhenJSR292Workload() { return _scratchSpaceFactorWhenJSR292Workload; }
 
 #if defined(J9VM_OPT_JITSERVER)
-   static int32_t getScratchSpaceFactorWhenJITServerWorkload() { return 2; }
-#endif
+   static int32_t _scratchSpaceFactorWhenJITServerWorkload;
+   static int32_t getScratchSpaceFactorWhenJITServerWorkload() { return _scratchSpaceFactorWhenJITServerWorkload; }
```

Comment:
I cannot find where `getScratchSpaceFactorWhenJITServerWorkload` is used.

Reply:
https://github.com/eclipse-openj9/openj9/blob/bdf0b3d46fdf55eb0507a304e85e5f54b8b86628/runtime/compiler/control/CompilationThread.cpp#L8727